src/test/kotlin/interactor/GetCityHasCheapestInternetConnectionInteractorTest.kt

```diff
@@ -39,7 +39,7 @@ internal class GetCityHasCheapestInternetConnectionInteractorTest {
     }
 
     @Test
-    fun `should return null when all city has null internet price`() {
+    fun `should return exception when all city has null internet price`() {
```

Comment:
good, but now change your way that naming the test function and make it like other members of team